jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt

```diff
@@ -9,12 +10,12 @@ import software.aws.toolkits.resources.message
 import javax.swing.JComponent
 
 class OutputsTableView : View, OutputsListener, Disposable {
-    private val table = DynamicTableView<Output>(
-        DynamicTableView.Field(message("cloudformation.stack.outputs.key")) { it.outputKey() },
-        DynamicTableView.Field(message("cloudformation.stack.outputs.value")) { it.outputValue() },
-        DynamicTableView.Field(message("cloudformation.stack.outputs.description")) { it.description() },
-        DynamicTableView.Field(message("cloudformation.stack.outputs.export")) { it.exportName() }
-    ).apply { component.border = JBUI.Borders.empty() }
+    private val key = DynamicTableView.Field<Output>(message("cloudformation.stack.outputs.key")) { it.outputKey() }
```

Comment:
Is this just for code-style? These no longer need to be broken out right?

Reply:
Yeah this change can be reverted now, will do